docs/visibility-on-elasticsearch.md

```diff
@@ -0,0 +1,89 @@
+# Overview
+Cadence visibility APIs allow users to list open or closed workflows with filters such as WorkflowType or WorkflowID.   
+With Cassandra, there are issues around scalability and performance, for example: 
+ - list large amount of workflows may kill Cassandra node.
+ - data is partitioned by domain, which means writing large amount of workflow to one domain will cause Cassandra nodes hotspots.
+ - query with filter is slow for large data.
+
+(With MySQL, there might be similar issues but not tested)
+
+In addition, Cadence want to support users to perform flexible query with multiple filters on even custom info.  
+That's why Cadence add support for enhanced visibility features on top of ElasticSearch (Note as ES below), which includes: 
+ - new visibility APIs to List/Scan/Count workflows with SQL like query
+ - search attributes feature to support users provide custom info
+ 
+# Quick Start
+## Local Cadence Docker Setup
+1. Increase docker memory to higher 6GB. Docker -> Preference -> advanced -> memory limit
+2. Get docker compose file. Run `<code>curl -O [https://raw.githubusercontent.com/uber/cadence/master/docker/docker-compose-es.yml](https://raw.githubusercontent.com/uber/cadence/master/docker/docker-compose-es.yml)</code>`
```

Comment:
Get rid of the `<code>` tags. Also, get rid of the link. It should just be:

> Run `curl -O https://raw.githubusercontent.com/uber/cadence/master/docker/docker-compose-es.yml`

You can also place this on it's own line by creating a code section with \`\`\`. That way, it's easier to copy.